develop/08_opendid/06_advanced.md

```diff
@@ -35,7 +35,7 @@ CLIENT_SPEC=$(cat <<EOF
 {
   "requirements": [{
     "cTypeHash":"0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac",
-    "trustedAttesters": [
+    "trustedIssuers": [
```

Comment:
this relates to a configuration of openDid and shouldn't be changed without changing the openDid code base first